config.example/env.sh

```diff
@@ -0,0 +1,6 @@
+# This file acts as a location to override the default configurations of deepops/scripts/*
+# Many of the scripts in this directory define global variables and set reasonable defaults
+# Global variables (in all caps) that are defined here will be automatically sourced and used in all scripts
+# See deepops/scripts/common.sh for implementation details
+
+DEEPOPS_VERSION="development-branch"
```

Comment:
This is a little confusing, does it set something to a development branch?

Reply:
This does nothing.  I was just adding an example variable and thought maybe we could put an actual version there to be imported in a future mechanism for easier debug.

Reply:
Do you mind commenting it out then? Maybe it's just me, but it looks like it's doing something

Reply:
Done.